public/css/gogs.css

```diff
@@ -1311,6 +1311,8 @@ footer .ui.language .menu {
 .repository.file.list #file-content .code-view .lines-num .hljs li,
 .repository.file.list #file-content .code-view .lines-code .hljs li {
   padding-left: 5px;
+  display: inline-block;
```

Comment:
Because the code is inside `pre` and `code`, when I add the newlines so highlightjs doesn't treat it as one massive block two are rendered (one for the `\n` and one for the `li`). This essentially removes the newline from the `li` so only one is rendered and it looks right. Furthermore the `width: 100%` fixes up the `li` so it is the with of the surrounding code box rather than just the actual code text.

I'll fix up the indent now.